SS_readcontrol_330.tpl

```diff
@@ -2944,7 +2944,7 @@
   // 5:  0/1 to float
   
   //  read setup and get the parameter count
-  echoinput << "# read Q setup only for fleets with survey/CPUE/effort observations, end with fleet_ID<0 "
+  echoinput << "# read Q setup only for fleets with survey/CPUE/effort observations, end with fleet_ID<0 " << endl
```

Comment:
Should all lines have a ";" at the end? I see that most lines do but not all of them so I'm not sure if it has to have it or should have it/